packages/react-openapi/src/OpenAPIOperation.tsx

```diff
@@ -29,7 +29,7 @@ export function OpenAPIOperation(props: {
 
     return (
         <div className={clsx('openapi-operation', className)}>
-            <div className="openapi-summary">
+            <div className="openapi-summary" id={operation.summary ? undefined : context.id}>
```

Comment:
Why only when `operation.summary` is undefined?

Reply:
Because else it's on the heading.